firedrake/slate/slac/compiler.py

```diff
@@ -610,7 +602,7 @@ def metaphrase_slate_to_cpp(expr, temps, prec=None):
     """
     # If the tensor is terminal, it has already been declared.
     # Coefficients in action expressions will have been declared by now,
-    # as well as any inverses with high reference count.
+    # as well as any other nodes with high reference count.
```

Comment:
What is "high"?

Reply:
Well it's a value currently set to 1. Meaning nodes with a reference count greater than 1 will have a temporary created for it. Whether this is the best way to do this sort of auxiliary temp creation is still open for discussion I think. This seems to work just fine based on my experiments.